core/controllers/profile.py

```diff
@@ -281,12 +281,17 @@ def post(self):
         """Handles POST requests."""
         username = self.payload.get('username')
         agreed_to_terms = self.payload.get('agreed_to_terms')
+        default_dashboard = self.payload.get('default_dashboard')
         can_receive_email_updates = self.payload.get(
             'can_receive_email_updates')
 
         has_ever_registered = user_services.has_ever_registered(self.user_id)
         has_fully_registered = user_services.has_fully_registered(self.user_id)
 
+        # Set the default dashboard for new users.
```

Comment:
Also, thinking defensively, could you add a "choices" property to core/storage/user/gae_models.py to restrict the possible values, or some kind of validation in update_user_default_dashboard, or here? Basically, somewhere along the backend pipeline, we should make sure that only allowed values make it through the flow.